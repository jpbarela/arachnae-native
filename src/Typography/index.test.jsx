// @flow
import * as React from "react";
import { render } from "@testing-library/react-native";
import { Title } from "./index";

test("Body renders", () => {
  const container = render(<Title>My Title</Title>);

  expect(container).toMatchSnapshot();
});

test("Title renders", () => {
  const container = render(<Title>My Title</Title>);

  expect(container).toMatchSnapshot();
});
