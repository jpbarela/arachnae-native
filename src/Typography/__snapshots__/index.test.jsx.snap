// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders 1`] = `
<Text
  style={
    Array [
      Object {
        "fontFamily": "arial",
      },
      Object {
        "fontSize": 48,
        "fontWeight": "bold",
      },
      Object {
        "color": undefined,
      },
    ]
  }
>
  My Title
</Text>
`;
